utilities_common/bgp_util.py

```diff
@@ -227,21 +229,30 @@ def get_bgp_summary_from_all_bgp_instances(af, namespace, display):
 
     bgp_summary = {}
     cmd_output_json = {}
+
     for ns in device.get_ns_list_based_on_options():
+        has_bgp_neighbors = False
         cmd_output = run_bgp_show_command(vtysh_cmd, ns)
+        device.current_namespace = ns
         try:
             cmd_output_json = json.loads(cmd_output)
+            if cmd_output_json:
+                has_bgp_neighbors = True
         except ValueError:
             ctx.fail("bgp summary from bgp container not in json format")
 
-        # exit cli command without printing the error message
-        if key not in cmd_output_json:
-            click.echo("No IP{} neighbor is configured".format(af))
-            exit()
+        # no bgp neighbors found so print basic device bgp info
+        if key not in cmd_output_json or not has_bgp_neighbors:
+            vtysh_cmd = "show ip bgp json"
```

Comment:
set the has_bgp_neighbors when the cmd_output_json is not as expected?

Reply:
done